standalone/Jenkinsfile

```diff
@@ -150,6 +154,32 @@ pipeline {
                 '''
             }
         }
+        stage('Validate images') {
+            steps {
+                sh '''
+                    oc describe pod -l jaeger-infra=query-pod | grep "Image:"
+                    oc describe pod -l jaeger-infra=collector-pod | grep "Image:"
+                    oc describe pod --selector='job-name=jaeger-standalone-performance-tests' | grep "Image:"
+                '''
```

Comment:
Will this eventually include some automated check to extract the image name and compare to expected?